charts/openstack-cluster/values.yaml

```diff
@@ -290,10 +290,10 @@ controlPlane:
     enabled: true
     # The spec for the health check
     spec:
-      # By default, unhealthy control plane nodes are always remediated
-      maxUnhealthy: 100%
-      # If a node takes longer than 10 mins to startup, remediate it
-      nodeStartupTimeout: 10m0s
+      # By default, don't remediate control plane nodes when more than one is unhealthy
+      maxUnhealthy: 1
```

Comment:
Maybe lets do 40% here as well?

Reply:
... just trying to convince myself that this will allow one node to be fixed, I guess it does.